be/src/runtime/time_types.cpp

```diff
@@ -298,31 +302,25 @@ bool date::from_string(const char* date_str, size_t len, int* year, int* month,
     return true;
 }
 
-// Get date base on format "%Y-%m-%d", '-' means any char.
+// Get date base on format "%YYYY-%mm-%dd", where '-' means any char.
```

Comment:
`%Y` is correct, it's different from `YYYY`

Reply:
Fixed.